src/components/Tooltip/tests/Tooltip.test.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Tooltip from '../Tooltip';
+import Link from '../';
+
+
+describe('Tooltip Component Suit', () => {
+    describe('Test children property', () => {
+        it('should verify children property is defined as link', () => {
+            const tooltipWrapper = mount(
```

Comment:
same as previous comment. I would push line 11 up to line 10 and take out the comma at the end of line 13